containers/payments/subscription/CustomProfessionalSection.js

```diff
@@ -7,20 +7,18 @@ import { range } from 'proton-shared/lib/helpers/array';
 import PlanPrice from './PlanPrice';
 import { getTextOption, getPlan, getAddon } from './helpers';
 
-const memberOptions = range(1, 5001).map((value, index) => ({
-    text: getTextOption('member', value, index),
-    value: index
-}));
-
-const domainOptions = range(2, 101).map((value, index) => ({
-    text: getTextOption('domain', value, index),
-    value: index
-}));
-
 const CustomProfessionalSection = ({ plans, model, onChange }) => {
     const professionalPlan = getPlan(plans, { name: 'professional' });
     const memberAddon = getAddon(plans, { name: '1member' });
     const domainAddon = getAddon(plans, { name: '1domain' });
+    const memberOptions = range(1, 5001).map((value, index) => ({
```

Comment:
Not related to your PR but @EpokK should this be a select with 5000 options?

Reply:
We need to support existing options on v3:
![Capture d’écran 2019-05-28 à 16 26 12](https://user-images.githubusercontent.com/1345786/58485983-53da3480-8165-11e9-9a4f-ff48d48e3b93.png)

Reply:
input number ? here if the step is 1 it's... "sympa"